format/spec.md

```diff
@@ -1084,14 +1100,16 @@ The 32-bit hash implementation is 32-bit Murmur3 hash, x86 variant, seeded with
 | **`uuid`**         | `hashBytes(uuidBytes(v))`		[4]      | `f79c3e09-677c-4bbd-a479-3f349cb785e7` ￫ `1488055340`               |
 | **`fixed(L)`**     | `hashBytes(v)`                            | `00 01 02 03` ￫ `-188683207`               |
 | **`binary`**       | `hashBytes(v)`                            | `00 01 02 03` ￫ `-188683207`               |
+| **`geometry`**     | `hashBytes(wkb(v))` [5]                   | `(1.0, 1.0)` ￫ `-246548298`              |
```

Comment:
I would probably _not_ specify how to hash geometry because we don't yet know how to do it correctly. The reason why we have the second table (hash requirements that are not part of bucket) is that we don't want anyone to forget that float and double should hash to the same value.

Reply:
@szehon-ho, I don't think we should specify this or allow geometry in bucket transforms because of issues with equality.